rollup.config.mjs

```diff
@@ -124,14 +124,6 @@ export default [
 								fs.writeFileSync('src/compiler/compile/internal_exports.ts', `// This file is automatically generated\nexport default new Set(${JSON.stringify(mod.exports)});`);
 							}
 						}
-
-						fs.writeFileSync(`${dir}/package.json`, JSON.stringify({
-							main: './index',
-							module: './index.mjs',
-							types: './index.d.ts'
-						}, null, '  '));
-
-						fs.writeFileSync(`${dir}/index.d.ts`, `export * from '../types/runtime/${dir}/index';`);
```

Comment:
We can't get rid of this line because else TypeScript without `moduleResolution: bundler` won't find the type definitions anymore.